docs/spi_driver.md

```diff
@@ -28,17 +45,17 @@ Initialize the SPI driver. This function must be called only once, before any of
 
 ---
 
-### `void spi_start(pin_t slavePin, bool lsbFirst, uint8_t mode, uint8_t divisor)`
+### `bool spi_start(pin_t slavePin, bool lsbFirst, uint8_t mode, uint16_t divisor)`
 
 Start an SPI transaction.
 
 #### Arguments
 
- - `pin_t slavePin`  
+ - `pin_t slavePin`
```

Comment:
Please keep the double spaces at the end here, otherwise it all gets rendered on one line.

Reply:
Hmmm. Other docs pages I've done haven't had this issue -- is there something specific?
What circumstances cause the run-on?

Reply:
With:
![image](https://user-images.githubusercontent.com/4781841/78982822-53247400-7b66-11ea-8ef4-7ea36af3d32b.png)
Without:
![image](https://user-images.githubusercontent.com/4781841/78982835-591a5500-7b66-11ea-89ea-3d7d23e8d261.png)
I just think it looks a little bit prettier :)